doc/UsersGuide/BuildingRunningTesting/SRW-SD.rst

```diff
@@ -84,21 +81,23 @@ Users will need to configure their experiment by setting parameters in the ``con
    
 Users will need to change the ``ACCOUNT`` variable in ``config.yaml`` to an account they have access to. They may also wish to adjust other experiment settings. For more information on each task and variable, see :numref:`Section %s <ConfigWorkflow>`. 
 
-On Level 1 systems, users can find :term:`ICs/LBCs` for the SRW-SD sample case in the usual :ref:`input data locations <Data>` under ``RAP/2019072200``. Users will need to add the following lines to ``task_get_extrn_*:`` in their ``config.yaml`` file, adjusting the file path to point to the correct data locations:
+On Level 1 systems, users can find :term:`ICs/LBCs` for the SRW-SD sample case in the usual :ref:`input data locations <Data>` under ``RAP/2019072200``. Users will need to switch the data directory for :term:`ICs/LBCs` if running on Orion or Hercules, on the following lines in ``config.yaml/task_get_extrn_*:``:
```

Comment:
If we use the locations below, we should probably change this line because those are not the usual input data locations.

Reply:
I will update the line to remove the "usual location" bit. Good catch.